polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -65,8 +68,12 @@ export interface TooltipProps {
   onOpen?(): void;
   /* Callback fired when the tooltip is dismissed */
   onClose?(): void;
+  /** Subdued text rendered after the tooltip content */
+  suffix?: string;
```

Comment:
I would prefer not to add a property when `children` already has the ability to have react children. This should be guidance if we want to recommend people to do subdued text after the tooltip not a property with documentation etc.

Reply:
@alex-page Makes sense to me – I'll keep the new addition to the polaris.shopify.com site, but it'll just show how to add a suffix using the `content` prop instead.

Reply:
Lovely, thanks @mrcthms